allure-report-face/src/components/widgets-grid/WidgetsGridView.js

```diff
@@ -7,7 +7,9 @@ import {className} from '../../decorators';
 import allurePlugins from '../../pluginApi';
 
 const widgetTpl = (id) => `<div class="widget island" data-id="${id}">
-    <div class="widget__handle fa fa-arrows"></div>
+    <div class="widget__handle">
+        <span class="fa fa-ellipsis-v"></span> <span class="fa fa-ellipsis-v"></span>
```

Comment:
All dotted examples here are for list views. In list views I also use dotted area (in can be done with plain css, see how it is working in testpalm). But there is not a list item, so I think drag and drop icon is better

![image](https://cloud.githubusercontent.com/assets/812240/15568849/58d2c0ee-232e-11e6-94b0-11f2ee967f95.png)